docs/introduction/en/basics.md

```diff
@@ -0,0 +1,109 @@
+* `codeHash`: The hash of the EVM code that gets executed when address receives a message call. Unlike other fields, it
+is immutable once established. All such code fragments are stored in a state database for later retrieval. 
+* `stakingBalance`: A scalar value equals to the number of staked Drip. 
+* `storageCollateral`: A scalar value equals to the number of Drip used as collateral for storage, which will be returned to
+balance if the corresponding storage is released. 
+* `accumulatedInterestReturn`: A scalar value equals to the number of Drip in accumulated interest return. 
+* `admin`: The address of the administrator
+* `sponsorInfo`: The sponsor information contains five components: `sponsor for gas`, `sponsor for collateral`, `sponsor gas
+limit`, `sponsor balance for gas` and `sponsor balance for collateral`. 
+
+
+## Transactions
+A Conflux transaction is a single instruction composed by an external actor with a Conflux account, and this instruction is cryptographically signed under the associated private key of the sending account. The authentication key, i.e. the sending account’s associated public key, is also included in the transaction for verification.
+
+### Transaction Types
+There are two types of transactions depending on the “destinations”:
+1. to an account address: these are normal transactions that may transfer value and/or result in message calls, known as
+action transactions;
+2. to “nowhere”: these transactions are used to create new contracts, known as contract creation transactions.
+
+### Transaction fields:
+Both types of transactions share the following common fields:
+* `from`: A address indicating the transaction sender.
+* `to`: A address indicating the transaction receiver.
+* `nonce`: A scalar value equal to the number of previously sent transactions.
+* `gasPrice`: A scalar value indicating the number of Drip to be paid per unit of gas that is consumed as a result of the
+execution of transaction.
+* `gas`: A scalar value indicating the total amount of gas paid for the cost of the execution of transaction. This is paid up-front,
+before any actual computation is done, and may not be increased or refunded later. It is
+the transaction sender’s responsibility to avoid any extravagance caused by an unnecessarily high gasLimit.
+* `value`: A scalar value equal to the amount of Drip that the transactions sender wants to transfer to the recipient, i.e. the
+account specified in transaction or the newly created contract. 
+* `storageLimit`: A scalar value indicating the maximum increment of storage used in the execution of transaction, measured in
+bytes.
+* `epochHeight`: A scalar value specifying the range of epochs where transaction can be executed. Such that transaction can only be executed between the epochs of [Te − 100000, Te + 100000].
+* `chainId`: A binary chain id indicating where transaction is intended to be executed. 
```

Comment:
A scalar value indicating the ID of the chain where the transaction is intended to be executed.